Solutions/Marain.UserNotifications.Specs/Marain.UserNotifications.Specs.csproj

```diff
@@ -14,28 +14,39 @@
     -->
     <NoWarn>SA0001;SA1204;SA1600;SA1602;CS1591</NoWarn>
     <RootNamespace>Marain.UserNotifications.Specs</RootNamespace>
+
+  </PropertyGroup>
+  <PropertyGroup>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+    <RestoreLockedMode Condition="$(ContinuousIntegrationBuild) == 'true'">true</RestoreLockedMode>
   </PropertyGroup>
   <ItemGroup>
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\ApiDeliveryChannelClient\MarkNotificationAsRead.feature.cs" />
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\ManagementApiClient\CreateNotifications - Copy.feature.cs" />
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\ManagementApi\CreateNotifications - Copy.feature.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Corvus.Configuration.TestEnvironment" Version="1.2.3" />
-    <PackageReference Include="Corvus.Testing.AzureFunctions.SpecFlow" Version="1.4.2" />
-    <PackageReference Include="Endjin.RecommendedPractices" Version="1.2.0">
+    <PackageReference Include="Corvus.Configuration.TestEnvironment" Version="1.2.5" />
+    <PackageReference Include="Corvus.Testing.AzureFunctions.SpecFlow" Version="1.5.0-253-unix-support.5" />
```

Comment:
This looks like another "preview button was checked in the NuGet package manager when doing an update". This preview is actually quite old because we've not yet had time to return the work on the branch in question. So it's a preview that's less up to date than the current non-preview build!

Just take the latest non-preview version.